proxy/context.go

```diff
@@ -112,6 +112,8 @@ func DefaultMeshConfig() proxyconfig.ProxyMeshConfig {
 		DiscoveryAddress:   "istio-pilot:8080",
 		EgressProxyAddress: "istio-egress:80",
 
+		AccessLogFile:          "/dev/stdout",
+		ErrorLogFile:           "/dev/stderr",
```

Comment:
I don't think we need to redirect stderr - it is handled correctly by systemd, and goes where expected (journal).
Also AFAIK stderr doesn't show up on envoy config, and envoy doesn't use it.

The access logs are the only problem.